src/main/java/application/HandlerHelper.java

```diff
@@ -0,0 +1,14 @@
+package main.java.application;
+
+import main.java.server.request.Request;
+import main.java.server.response.Response;
+import main.java.server.response.ResponseBuilder;
+
+class HandlerHelper {
+
+    static Response assembleResponse(Request request, String body) {
+        ResponseBuilder builder = new ResponseBuilder();
+        return builder.build(request, body, Routes.getROUTES());
```

Comment:
Does this implementation mean that we're calling `Router.getROUTES` on every request?

Reply:
Yes - probably makes more sense to extract the corresponding `methods/handler` hash before calling `builder.build` and then just pass that in.